server/statistics/store_hot_region_infos.go

```diff
@@ -0,0 +1,10 @@
+package statistics
+
+// StoreHotRegionInfos : used to get human readable description for hot regions.
+type StoreHotRegionInfos struct {
```

Comment:
I prefer to merge this file into `hot_regions_stat.go`

Reply:
These two struct are too simple, and not use by other types or functions in this package. So I'm thinking about moving them to where they are used.
Is there any suggestions?